client/src/i18n/en/tree.json

```diff
@@ -2,6 +2,7 @@
   "TREE": {
     "ACCOUNT": "Account Management",
     "ACCOUNT_STATEMENT": "Account Statement",
+    "ACCOUNT_CONFIGURATION":"Account Configuration",
```

Comment:
Could you call the key `PAYROLL_ACCOUNT_CONFIGURATION`?  We might get confused about what it is for otherwise...